docs/UserGuide.md

```diff
@@ -423,7 +428,7 @@ Format: `editOrder <INDEX> [by/DEADLINE] [c/PRICE] [d/DESCRIPTION] [s/STATUS]`
   The index **must be a positive integer** 1, 2, 3, …​
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
-* There are 3 different status:
+* There are 3 different status (they are all case-insensitive):
```

Comment:
As long as this is true

Reply:
yes, it accept both "PENDING" and "pending"